CONTRIBUTING.md

```diff
@@ -114,52 +114,43 @@ The administration panel should now be available at http://localhost:4000/admin.
 - `yarn test:front` runs front-end related tests.
 - `yarn test:front:watch` runs an interactive test watcher for the front-end.
 - `yarn test:unit` runs the back-end unit tests.
-- `yarn test:e2e` runs an end-to-end test suite.
+- `yarn test:api` runs the api integration tests.
 - `yarn test:generate-app` generates a test application.
 - `yarn test:start-app` starts the test application.
 
 ---
 
-## Running the End-To-End (e2e) tests
+## Running the API Integration tests
 
-The end-to-end tests require a Strapi app to be able to run. You can generate a "test app" using `yarn test:generate-app <database>`:
+The API integration tests require a Strapi app to be able to run. You can generate a "test app" using `yarn test:generate-app <database>`:
 
 ```bash
 $ yarn test:generate-app sqlite
 $ yarn test:generate-app postgres
 $ yarn test:generate-app mysql
 ```
 
-A new app is required every time you run the end-to-end tests otherwise, the test suite will fail. A script is available to make this process easier: `node test/e2e.js`. It will delete the current test app, generate a new one and run the test suite.
+A new app is required every time you run the API integration tests otherwise, the test suite will fail. A command is available to make this process easier: `yarn test:api`.
```

Comment:
```suggestion
A new app is required every time you run the API integration tests, otherwise the test suite will fail. A command is available to make this process easier: `yarn test:api`.
```